tools/static-assets/server/boot.js

```diff
@@ -285,7 +285,19 @@ var loadServerBundles = Profile("Load server bundles", function () {
           }
         }
 
-        var resolved = require.resolve(name);
+        try {
+          var resolved = require.resolve(name);
+        } catch (err) {
+          const nameParts = name.split("/");
+          // If it looks like `meteor/xxx`, the user may forgot to add the
+          // package before importing it.
+          if (nameParts.length === 2 && nameParts[0] === "meteor") {
```

Comment:
This is just personal preference (totally optional - just thought I'd throw it in); when working with `split` I like using array destructuring to give the split up array parts nice meaningful names (makes the code more readable, easier to maintain, etc.). So you could consider adjusting your `split` approach to something like:

```js
...
} catch (error) {
  if (name && name.startsWith('meteor/')) {
    const [meteorPrefix, packageName] = name.split('/');
    throw new Error(
      `Cannot find package "${packageName}". ` +
      `Try "meteor add ${packageName}".`
    );
  } else {
    throw error;
  }
}
...
```